docs/src/modules/java/pages/developer-tools.adoc

```diff
@@ -0,0 +1,104 @@
+= Developer Tools
+
+include::ROOT:partial$include.adoc[]
+
+The Java tooling is published to Lightbend's public https://cloudsmith.io/~lightbend/repos/akkaserverless/packages/[Maven repository]. 
+
+== Kickstart
+
+A https://maven.apache.org/guides/introduction/introduction-to-archetypes.html[Maven archetype]; _maven-archetype-akkasls_ provides the Kickstart tooling. You supply details such as the desired artifact and group IDs, and the archetype provides a new Maven project directory with the Akka Serverless SDK and associated development support tooling set up.
+
+The archetype is located at Lightbend's Maven repository. In order to use it, your global https://maven.apache.org/settings.html[Maven settings] must be configured to include this repository. Your active profile must be updated to include the following repository definition:
+
+[source,xml]
+----
+<repository>
+  <id>lightbend-akkaserverless</id>
+  <name>repo-lightbend-com-akkaserverless</name>
+  <url>https://repo.lightbend.com/lightbend/akkaserverless/</url>
+  <releases>
+    <enabled>true</enabled>
+    <updatePolicy>always</updatePolicy>
+  </releases>
+  <snapshots>
+    <enabled>true</enabled>
+    <updatePolicy>always</updatePolicy>
+  </snapshots>
+</repository>
+----
+
+With this configuration in place, the archetype can be used via the `mvn` CLI.
```

Comment:
Same here. No need to add anything to the settings file. Users can use the archetypes directly from Maven Central.